docs/core/entity/climate.md

```diff
@@ -29,6 +29,8 @@ Properties should always only return information from memory and not do I/O (lik
 | preset_modes            | <code>list[str] &#124; None</code>  | **Required by SUPPORT_PRESET_MODE**  | The available presets.                                                     |
 | swing_mode              | <code>str &#124; None</code>        | **Required by SUPPORT_SWING_MODE**   | The swing setting.                                                         |
```

Comment:
maybe also update `swing_mode` description to state that if the entity provides horz swing, then this is used for vertical.